gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -81,7 +86,12 @@ def test_{{ service.client_name|snake_case }}_from_service_account_info():
         {% if service.host %}assert client.transport._host == '{{ service.host }}{% if ":" not in service.host %}:443{% endif %}'{% endif %}
 
 
-@pytest.mark.parametrize("client_class", [{{ service.client_name }}, {{ service.async_client_name }}])
+@pytest.mark.parametrize("client_class", [
+    {{ service.client_name }},
+    {%- if 'grpc' in opts.transport %}
```

Comment:
This construct is used everywhere in the template. Since it is so commonly used, please consider simplifying it by introducing some helper method, such that it look something like:
```
{%- if grpc %}
stuff
{%- endif %}